docs/micropython.md

```diff
@@ -25,7 +26,8 @@ Module                          Support status
 ==============================  =================
 
 .. [1] Untested/low-priority, but some of it might work.
-.. [2] It might work, but isn't useful without ``ev3dev2.display``.
+.. [2] Display() isn't implemented. Use ``ev3dev2.console`` for text-only, using ANSI codes to the EV3 LCD console.
+.. [3] It might work, but isn't useful without ``ev3dev2.display``.
```

Comment:
Maybe we should just officially declare `ev3dev2.fonts` as not supported for micropython. That might be more clear than our current wording.

Reply:
Done!